README.md

```diff
@@ -9,27 +9,28 @@ This [Prover](https://github.com/kroma-network/kroma/blob/dev/specs/zkevm-prover
 - Golang (go1.20.3)
 - ProtoBuf (libprotoc 3.21.12)
 
-## Setup
-
-```shell
-> git clone https://github.com/kroma-network/kroma-prover.git
-> cd kroma-prover
-> git submodule update --init
-```
-
 ## Kroma Prover Binary
 
-Prover server (entry: prover-grpc/src/prover_server.rs)
+Prover server (entry: prover-server/src/server_main.rs)
 
 ```shell
 # build
 > cargo build --release --bin prover-server
 
 # run
-> ./target/release/prover-server --config <config-file-path>
+> ./target/release/prover-server --ip "127.0.0.1:3030"  
 ```
+Mock Prover server (which always return zero proof for test)
+```shell
+# build 
+> cargo build --release --bin prover-server --features mock-server
 
-Mock client for test (entry: prover-grpc/src/client_mock.rs)
+# run
+> ./target/release/prover-server --ip "127.0.0.1:3030"
+```
+
+
+Mock client for test (entry: prover-grpc/src/mock_client.rs)
 
 ```shell
 # build
```

Comment:
`cargo build --release --bin client-mock` doesn't work.
i had to use  `cargo build --release --bin mock-client`